build_scripts/build_runner.py

```diff
@@ -784,6 +814,15 @@ def main():
     dictConfig(LOG_CONFIG)
 
     try:
+        if not args.changed_repo and not args.sources:
+            log.error('"--changed-repo" or "--sources" argument bust be added')
+            exit(Error.CRITICAL.value)
+        elif not args.changed_repo and args.sources and args.stage in [Stage.COPY, Stage.PACK, Stage.INSTALL]:
+            log.error('Only "clean", "extract" and "build" stages available for running with "--sources" argument')
+            exit(Error.CRITICAL.value)
+        elif args.changed_repo and args.sources:
+            log.info('Priority for --changed-repo, not for --sources')
```

Comment:
Warning: The --sources argument is ignored because the --changed-repo is set
or it can be an error to prevent possible usability issues.